easybuild/easyconfigs/o/ORFfinder/ORFfinder-0.4.3.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PackedBinary'
+
+name = 'ORFfinder'
+version = '0.4.3'
```

Comment:
@hajgato Is it worth mentioning how you determined the version in a comment (since the tarball itself is versionless)?

Reply:
@boegel Done.